ajax/libs/OwlCarousel2/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "OwlCarousel2",
+  "filename": "owl.carousel.min.js",
+  "description": "Touch enabled jQuery plugin that lets you create beautiful responsive carousel slider.",
+  "version": "2.0.0-beta.3",
+  "homepage": "https://github.com/smashingboxes/OwlCarousel2",
+  "author": {
+    "name": "Smashing Boxes",
+    "email": "opensource@smashingboxes.com",
+    "url": "http://owlcarousel.owlgraphic.com"
+  },
+  "license": "MIT",
+  "keywords": [
+    "responsive",
+    "carousel",
+    "owlcarousel",
+    "jQuery",
+    "plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/smashingboxes/OwlCarousel2"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/smashingboxes/OwlCarousel2.git",
+    "basePath": "dist",
+    "files": [
+      "**/*.+(js|css|png|gif)"
```

Comment:
As we can ignore `*.md` and `LICENSE` files, it is fine to use `"**/*"` here ~
Please change to `"**/*"`.